learn/cookbooks/azure.md

```diff
@@ -0,0 +1,78 @@
+# Deploy a Meilisearch instance on Azure
+
+_The following is a guest post by Christopher Maneu, Data Engineering Lead Advocate at Microsoft._
+
+## Introduction
+
+In this tutorial, we will see how to deploy a Meilisearch instance on Azure, and also some considerations for hosting a production Meilisearch instance on Azure.
+
+::: tip
+If you don't have yet an Azure subscription, you can [create one here](https://azure.microsoft.com/free/). With the Azure Free tier, you can run a Meilisearch instance for free for at least 12 months.
+:::
+
+## One-Click Deploy
+
+You can deploy a Meilisearch instance via the official [Meilisearch Docker image](https://hub.docker.com/r/getmeili/meilisearch) by clicking the following button:
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fcmaneu%2Fmeilisearch-on-azure%2Fmain%2Fmain.json).
+
+![The Azure portal prompting for information to deploy Meilisearch](/azure/01.azure-deploy-button.png)
+
+When clicking this button, you'll be redirected to the Azure Portal, and asked a few questions:
```

Comment:
```suggestion
When clicking this button, you'll be redirected to the Azure Portal and asked a few questions:
```

Reply:
Our internal style tool always yell at me when I don't put a comma before an and. So in English, I add them "everywhere". Fact is it's not [right](https://www.grammarly.com/blog/comma-before-and/). Thanks for this discovery ;)